### ISSUE TITLE
Buffer should return cartodb_id

### ISSUE THREAD

**Issue by rochoa** [CartoDB/camshaft#212]
`buffer` does not return `cartodb_id` when using `dissolved` option.

**Comment by saleiva** [comment]
hey @dgaubert @javisantana is there any chance we can prioritize this? Getting a few requests lately

**Comment by javisantana** [comment]
this is the kind of analysis which is really dangerous: a input attribute can modify the schema (remove a column) and that can break all the analysis chain.

So we either add a cartodb_id when it's not dissolved (just one row but ok) or we keep like this

**Comment by saleiva** [comment]
so we add the cartodb_id, right?

**Comment by javisantana** [comment]
let's add always cartodb_id to fix the issue

**Comment by jgoizueta** [comment]
This is the kind of  query that hits the tiler in this case:

```
SELECT ST_AsTWKB(ST_Simplify(ST_RemoveRepeatedPoints("the_geom_webmercator",1e-05),1e-05,true),5) AS geom,"cartodb_id"
FROM (
  SELECT ST_Transform(the_geom, 3857) the_geom_webmercator
    FROM (
      select * from analysis_2f13a3dbd7_ffecdfb1e9d7e6eca24423afc8eb218c08391ff2
     ) _cdb_analysis_query
) as cdbq
WHERE "the_geom_webmercator" && ST_MakeEnvelope(-20037508.3,20037508.29613578,-20037508.29613578,20037508.3,3857)
```

The cached analysis table has no `cartodb_id`

**Comment by dgaubert** [comment]
This is already fixed in [0.47.0](https://github.com/CartoDB/camshaft/releases/tag/0.47.0). It is going to be deployed today.
